ignite/services/plugin/scaffold_test.go

```diff
@@ -1,45 +1,79 @@
-package plugin
+package plugin_test
 
 import (
 	"context"
 	"os"
 	"path/filepath"
 	"testing"
 
-	"github.com/ignite/cli/v28/ignite/pkg/gocmd"
 	"github.com/stretchr/testify/require"
 	"gopkg.in/yaml.v2"
+
+	"github.com/ignite/cli/v28/ignite/pkg/gocmd"
+	"github.com/ignite/cli/v28/ignite/services/plugin"
 )
 
+const fooBarAppURI = "github.com/foo/bar"
+
 func TestScaffold(t *testing.T) {
 	// Arrange
 	tmp := t.TempDir()
 	ctx := context.Background()
 
 	// Act
-	path, err := Scaffold(ctx, tmp, "github.com/foo/bar", false)
+	path, err := plugin.Scaffold(ctx, tmp, fooBarAppURI, false)
 
 	// Assert
 	require.NoError(t, err)
 	require.DirExists(t, path)
 	require.FileExists(t, filepath.Join(path, "go.mod"))
 	require.FileExists(t, filepath.Join(path, "main.go"))
+}
 
-	// app.ignite.yml check
-	appYML, err := os.ReadFile(filepath.Join(path, "app.ignite.yml"))
-	require.NoError(t, err)
-	var config AppsConfig
-	err = yaml.Unmarshal(appYML, &config)
-	require.NoError(t, err)
-	require.EqualValues(t, 1, config.Version)
-	require.Len(t, config.Apps, 1)
+func TestScaffoldedConfig(t *testing.T) {
+	// Arrange
+	ctx := context.Background()
+	path := scaffoldApp(t, ctx, fooBarAppURI)
+
+	// Act
+	cfg := readConfig(t, path)
+
+	// Assert
+	require.EqualValues(t, 1, cfg.Version)
+	require.Len(t, cfg.Apps, 1)
+}
 
-	// Integration test check
-	err = gocmd.Test(ctx, filepath.Join(path, "integration"), []string{
+func TestScaffoldedTests(t *testing.T) {
+	// Arrange
```

Comment:
I would delete those comments as they provide no benefit for the reader.

Reply:
Interesting, I had to check because I always used the three section comments as part of the AAA (Arrange-Act-Assert) testing pattern without really questioning if they are actually necessary.

There seems to be a consensus everywhere I look about explicitly using them, the comments add clarity on the pattern being used and also where each section begins and ends. I would prefer to keep them, but I don't have a strong opinion.

@julienrbrt would you let me know if the nit force is still strong :), if so I will remove them.

Reply:
IMO the spacing between each block makes it clear enough for readers to understand what each block is doing. But certainly it doesn't hurt to have comments.

Reply:
Yeah, it has no benefit thanks to the spacing. but I guess it's fine to keep it written down.